apps/ledger-live-desktop/tests/specs/speculos/receive.address.spec.ts

```diff
@@ -49,3 +61,31 @@ for (const account of accounts) {
     );
   });
 }
+
+test.describe("Receive", () => {
```

Comment:
same comment as for old PR - would it make sense to make it as a part of the first test?
old comment:
would it make sense to add this check to the main receive test case to avoid launching electron + speculos just for warning message verification?

Reply:
I don't think so as this scenario cannot go further in the flow (account has an empty balance and address isn't activated), whereas all TC gathered in the same describe share this flow and validate the address in the end